test-e2e/terraform_test.go

```diff
@@ -5,12 +5,14 @@ package test
 
 import (
 	"fmt"
+	"os"
 	"strings"
 	"testing"
 )
 
 // We're testing that we can download and run typical Terraform versions via ize
 func TestIzeTerraformVersion_1_0_10(t *testing.T) {
+	fmt.Println(os.Getenv("ACTIONS_RUNNER_DEBUG"))
 
```

Comment:
Seems like `RUNNER_DEBUG` is the current name of the variable.
`RUNNER_DEBUG=1` if debug is on
`RUNNER_DEBUG` is missing or not set to 1 if the debug is off.

Please update the code accordingly.